README.md

```diff
@@ -22,20 +22,21 @@ For VSCode, you should see a popup in the bottom right when you open the project
 
 ### Running the Dev Server
 
-To run the development server, you first need to have `npm`/`pnpm`/`yarn` installed in your system.
-Only `pnpm` is regularly used and tested for the site, so you might encounter bugs or incompatibilities using an alternate package manager. [Instructions on how to install `pnpm` can be found on its official installation page.](https://pnpm.io/installation)
+To run the development server, you first need to have `npm` installed in your system. Usually this comes pre-installed with [Node.js](https://nodejs.org), so you might want to install that first.
 
-Once `pnpm` is installed, clone this repository, navigate into the project folder and run `pnpm i` to install dependencies:
+##### Only `npm` is regularly used and tested for the site, so you might encounter bugs or incompatibilities using an alternate package manager.
```

Comment:
That's an unusual way to make bold text, and most likely not very good for accessibility tools - do we really not have better formatting means ?

Reply:
Wasnt even intended to be bold, haha.
Did it the proper™️ way now tho ^^